src/azure-cli/azure/cli/command_modules/acs/decorator.py

```diff
@@ -249,10 +279,12 @@ def get_dns_name_prefix(self, enable_validation: bool = False, **kwargs):
             mc_property = self.mc.dns_prefix
 
         # set defautl value
```

Comment:
typo defautl

Reply:
Will fix in another PR since before the change of this PR, the same typo already exists.